pom.xml

```diff
@@ -243,6 +218,12 @@ under the License.
       <version>${mavenVersion}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>1.7.30</version>
```

Comment:
1.7.29 max

Reply:
I removed this dependency as it added many unwanted logs.